test/docker/docker-compose.yaml

```diff
@@ -3,6 +3,7 @@ name: blip
 services:
     mysql57:
         image: mysql:5.7.34
+        platform: linux/amd64
```

Comment:
docker pull wasn't working in my M2 without these tags. Was getting the following error:

```
no matching manifest for linux/arm64/v8 in the manifest list entries
```

Adding the platform fixed it.